crypto/CMakeLists.txt

```diff
@@ -38,9 +38,39 @@ if (CONFIG_NRF_CC310_BL)
   target_link_libraries(${IMAGE}nrfxlib_crypto INTERFACE ${CC310_BL_LIB})
 endif()
 
+if (CONFIG_NRF_CC310_PLATFORM)
+  set (NRF_CC310_PLATFORM_BASE ${CMAKE_CURRENT_SOURCE_DIR}/nrf_cc310_platform)
+  set (NRF_CC310_PLATFORM_VER 0.9.0-rc1)
+  if (NOT CONFIG_CC310_INTERRUPTS_BACKEND)
+    set(CC310_PLATFORM_FLAVOR no-interrupts)
+  endif()
+  set(NRF_CC310_PLATFORM_LIB
+    ${NRF_CC310_PLATFORM_BASE}/${lib_path}/${CC310_PLATFORM_FLAVOR}/libnrf_cc310_platform_${NRF_CC310_PLATFORM_VER}.a
+  )
+
+  if (NOT EXISTS ${NRF_CC310_PLATFORM_LIB})
+    message(WARNING "This combination of SoC, floating point ABI, and interrupts settings is not supported by the nrf_cc310_platform lib."
+	            "(${NRF_CC310_PLATFORM_LIB} doesn't exist.)")
+  endif()
+  #
+  # Uses a normal add_library to avoid --whole-archive
+  #
+  add_library(${IMAGE}platform_cc310 STATIC IMPORTED GLOBAL)
+  set_target_properties(${IMAGE}platform_cc310 PROPERTIES IMPORTED_LOCATION
+	  		"${NRF_CC310_PLATFORM_LIB}")
+  target_include_directories(${IMAGE}platform_cc310 INTERFACE
+	  		"${NRF_CC310_PLATFORM_BASE}/include")
```

Comment:
I believe this line is redundant with line 65 and can be removed.

Reply:
Fixed (Removed line 65 instead)